lib/rubocop/cli/command/execute_runner.rb

```diff
@@ -61,6 +61,11 @@ def display_error_summary(errors)
         end
 
         def maybe_print_corrected_source
+          # Integration tools (like RubyMine) expect to have only the JSON result
+          # when specifying JSON format.
+          # See: https://github.com/rubocop-hq/rubocop/issues/8673
+          return if @options[:format] == 'json'
```

Comment:
Btw, shouldn't we do the same for the HTML formatter as well? Might be weird to append plain text to it.

Reply:
Sure! I think also JUnit formatter is similar, so I added HTML and JUnit formatters.